wiki/Configuration:-Miscellaneous.md

```diff
@@ -233,4 +235,13 @@ hotkey-overlay {
 }
 ```
 
+#### `hide-unbound`
+
+Set the `hide-unbound` flag if you don't want to see actions that haven't been bound.
```

Comment:
```suggestion
By default, niri will show the most important actions even if they aren't bound to any key, to prevent confusion.
Set the `hide-unbound` flag if you want to hide all actions not bound to any key.
```